hazelcast/src/main/java/com/hazelcast/quorum/QuorumEvent.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2008-2015, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.quorum;
+
+import com.hazelcast.core.Member;
+import java.util.Collection;
+import java.util.EventObject;
+
+/**
+ * Represents a cluster quorum event. Hold member list, quorum threshold and the quorum result.
+ */
+public class QuorumEvent extends EventObject {
```

Comment:
This is only for sending events to the quorum result listeners about quorum size is met or not.